packages/fields-document/src/DocumentEditor/index.tsx

```diff
@@ -249,10 +250,21 @@ export function DocumentEditor({
 
         <DocumentEditorEditable
           css={
```

Comment:
We missed this in design spec... can you please ensure the background of the document field input element is set to `#fafbfc` (or preferably the token equivalent) when the user is not interacting with the form. See the `text` field for a reference of desired behaviour.

This will ensure that style/behaviour is consistent with what we have in place now.

![image](https://user-images.githubusercontent.com/6447754/180121200-d05ad03e-97c1-4700-bdc3-a2669c45eeac.png)
![SCR-20220721-i6j](https://user-images.githubusercontent.com/6447754/180121227-97ee0e46-2acd-4727-a450-c9a68715e736.png)

Reply:
Added changes. Please review